docs/requirements/requirements.md

```diff
@@ -144,7 +175,9 @@ This document is in the making since we are still in the requirements engineerin
 - There must be a configurations file for designing the experiments. See R-U-003.
 - The RA must be sufficiently deterministic in order to allow the user to repeatedly carry out experiments.
 - Researchers want to do experiments using the RA.
-- Origin: Supervisors, Researchers
+- It must be effortless to use the system for experimentation. -- Researcher *Toni*
+- As a researcher, I want control over my experiments. The systems must allow me to have this control. -- Researcher *Toni*
```

Comment:
Is this similar to the controlled environment mentioned elsewhere?

Reply:
Yes and no. The controlled environment refers to the researcher being able to control the state of the system at any experiment's start. This allows for repeatability. That means a researcher can be sure that the system is in the same state at the beginning of each experiment. Apart from that, the stakeholder -- researcher Toni -- also said that in general he wants to be able to control everything about an experiment, from start to finish. Therefore he requires a system that supports that and configuration unlocks this potential.